spec/ghp_metadata_generator_spec.rb

```diff
@@ -18,10 +18,17 @@
       ENV['PAGES_REPO_NWO'] = nil
     end
 
-    it "raises a NoRepositoryError" do
-      expect(-> {
-        subject.send(:nwo, site)
-      }).to raise_error(Jekyll::GitHubMetadata::NoRepositoryError)
+    context "without a git nwo" do
+      it "raises a NoRepositoryError" do
+        expect(-> {
+          subject.send(:nwo, site)
+        }).to raise_error(Jekyll::GitHubMetadata::NoRepositoryError)
+      end
+    end
+
+    it "grabs the git nwo from an HTTPS url" do
+      subject.class.any_instance.stub(:git_remote_url) { "htps://github.com/foo/bar" }
```

Comment:
`allow(subject).to receive(:git_remote_url).and_return("https://github.com/foo/bar")`